tests/cli/ursula/test_stake_via_allocation_contract.py

```diff
@@ -535,7 +535,6 @@ def test_collect_rewards_integration(click_runner,
 
     # Collect Policy Reward
     collection_args = ('stake', 'collect-reward',
-                       '--mock-networking',
```

Comment:
I'm a little confused why this option is no longer needed, could you clarify the solution? 

Also, it was previously available to all cli commands as part of the `general_config`, but now it is specified as a `middleware` config option for specific characters - was it that it was not as general as we previously thought?

Reply:
As far as I understand, the `stake collect-reward` command does not use this option. Of all the characters, only Alice, Bob and Felix use it. Plus, my intention was to only keep output and logging-related options in general config.

Reply:
@cygnusv would know more about that test.

Many of the characters use the `MockRestMiddleware()` setup for non-CLI tests, so it's possible that the option is useful for CLI tests as well but just not being utilized based on the current tests written. In any case, it's probably fine for now 👍 , and can be moved later if deemed necessary.